### PR TITLE
Update douban extension

### PR DESCRIPTION
**Pull request by iiwenwen**
## Checklist

- [ ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/douban/src/search-users.tsx [hunk 1]
```diff
@@ -98,6 +99,11 @@ export default function Command() {
                   shortcut={{ modifiers: ["cmd", "shift"], key: "d" }}
                   onAction={() => setShowingDetail(!showingDetail)}
                 />
+                <Action.CopyToClipboard
+                  title="Copy As Markdown"
+                  content={`[${user.title}(豆瓣)](${user.url})`}
```
Comment (pernielsentikaer) [inline extensions/douban/src/search-users.tsx:104]: At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

File: extensions/douban/src/search-sites.tsx [hunk 2]
```diff
@@ -98,6 +99,11 @@ export default function Command() {
                   shortcut={{ modifiers: ["cmd", "shift"], key: "d" }}
                   onAction={() => setShowingDetail(!showingDetail)}
                 />
+                <Action.CopyToClipboard
+                  title="Copy As Markdown"
+                  content={`[${site.title}(豆瓣)](${site.url})`}
```
Comment (pernielsentikaer) [inline extensions/douban/src/search-sites.tsx:104]: At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

File: extensions/douban/src/search-music.tsx [hunk 3]
```diff
@@ -116,6 +117,11 @@ export default function Command() {
                   shortcut={{ modifiers: ["cmd", "shift"], key: "d" }}
                   onAction={() => setShowingDetail(!showingDetail)}
                 />
+                <Action.CopyToClipboard
+                  title="Copy As Markdown"
+                  content={`[${music.title}(豆瓣)](${music.url})`}
```
Comment (pernielsentikaer) [inline extensions/douban/src/search-music.tsx:122]: At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

File: extensions/douban/src/search-movies.tsx [hunk 4]
```diff
@@ -111,6 +112,11 @@ export default function Command() {
                   shortcut={{ modifiers: ["cmd", "shift"], key: "d" }}
                   onAction={() => setShowingDetail(!showingDetail)}
                 />
+                <Action.CopyToClipboard
+                  title="Copy As Markdown"
+                  content={`[${movie.title}(豆瓣)](${movie.url})`}
```
Comment (pernielsentikaer) [inline extensions/douban/src/search-movies.tsx:117]: At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

File: extensions/douban/src/search-groups.tsx [hunk 5]
```diff
@@ -98,6 +99,11 @@ export default function Command() {
                   shortcut={{ modifiers: ["cmd", "shift"], key: "d" }}
                   onAction={() => setShowingDetail(!showingDetail)}
                 />
+                <Action.CopyToClipboard
+                  title="Copy As Markdown"
+                  content={`[${group.title}(豆瓣)](${group.url})`}
```
Comment (pernielsentikaer) [inline extensions/douban/src/search-groups.tsx:104]: At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

File: extensions/douban/src/search-games.tsx [hunk 6]
```diff
@@ -99,6 +100,11 @@ export default function Command() {
                   shortcut={{ modifiers: ["cmd", "shift"], key: "d" }}
                   onAction={() => setShowingDetail(!showingDetail)}
                 />
+                <Action.CopyToClipboard
+                  title="Copy As Markdown"
+                  content={`[${game.title}(豆瓣)](${game.url})`}
```
Comment (pernielsentikaer) [inline extensions/douban/src/search-games.tsx:105]: At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

File: extensions/douban/src/search-books.tsx [hunk 7]
```diff
@@ -116,6 +117,11 @@ export default function Command() {
                   icon={Icon.AppWindowList}
                   onAction={() => setShowingDetail(!showingDetail)}
                 />
+                <Action.CopyToClipboard
+                  title="Copy As Markdown"
+                  content={`[${book.title}(豆瓣)](${book.url})`}
```
Comment (pernielsentikaer) [inline extensions/douban/src/search-books.tsx:122]: At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

**Comment by pernielsentikaer** [comment]
Hi @iiwenwen 👋 

Thanks for your contribution 🔥  Could you add a [CHANGELOG](https://developers.raycast.com/basics/prepare-an-extension-for-store#version-history) entry too?

@crazyczy could you check this

File: extensions/douban/src/search-users.tsx (hunk 1 above)
Comment (iiwenwen) [reply extensions/douban/src/search-users.tsx:104]: hello,recovered.

**Comment by iiwenwen** [comment]
@pernielsentikaer I add CHANGELOG.

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Would it be an idea to make some components that the commands are using? Seems like most of the code is just duplicated. 🙂

**Comment by iiwenwen** [comment]
@pernielsentikaer I’m sorry, I’m not a programmer. I can’t do what you said for the time being. I want to know, is there any other problem with this PR? There is still no merger.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**